Documentation/coding-guidelines/project-guidelines.md

```diff
@@ -89,10 +88,18 @@ All supported targets with unique windows/unix build for netcoreapp:
 <PropertyGroup>
 ```
 
-##Full build configuration
-The `BuildConfiguration` property can either be set directly via
-`/p:BuildConfiguration=[BuildConfiguration]` or will be defaulted based
-on where you are running the build. The default will look like `netcoreapp-[RunningOS]-Debug-[RunningProcessArch]`, example `netcoreapp-Windows_NT-Debug-x64`.
+##Options for building
+
+A full or individual project build is centered around BuildConfiguration and will be setup in one of the following ways:
+
+1) `$(BuildConfiguration)` can directly be passed to the build.
+2) `$(Configuration)` can be passed to the build and `$(BuildConfiguration)` will be set to `$(Configuration)-$(ArchGroup)`. This is a convinence mechanism primarily to help with VS support because VS uses the `Configuration` property for switching between various configurations in the UI.
```

Comment:
It is primarily about individual projects but we could make it work for the root builds if we treat it as local. However I will update this to point out Configuration is only interesting for individual projects.